### DISCUSSION TITLE
Can't log in with android 13 and bitwarden ("The operation failed for an unknown transient reason")

### DISCUSSION THREAD

**Discussion by axel37** [solokeys/solo2#180] [General]
Hi there,

I've just received my solokeys that I mainly plan to use with my password manager, Bitwarden.
(I'm posting here after being redirected by their tech support)

I was able to succesfully register and login with both my keys on my windows machine (through the web browser, web extension and desktop app).
However, I couldn't log in with my android phone (neither through usb-c nor nfc), as I kept getting the following message : 

> "Please make sure your default browser supports WebAuthn and try again. Unknown error : The operation failed for an unknown transient reason".

The support person from bitwarden claims they were able to reproduce the issue then suggested I post here.

Thanks !

___

*Edit : Still not working as of Feb. 26th 2024*

**Comment by msiqx9** [comment]
I have just succeeded to log into the Bitwarden app with the Solo v2 key in Android 13 by means of NFC, on the 2nd attempt.

**Reply by axel37** [reply]
Did you do anything specific to succeed ? Because I've made more than two attempts ^^

Perhaps there's been an update ?

Thanks for the feedback, at least we know it works for *some* users.

**Comment by msiqx9** [comment]
> Did you do anything specific to succeed ? Because I've made more than two attempts ^^

I did not do anything special.  When Android 13 first came out I had [problems](https://github.com/solokeys/solo2/discussions/163#discussioncomment-6219672) with both my Solo and Yubico keys but over time with Android 13 updates in the last 6 months, this seems to have been resolved.

**Reply by axel37** [reply]
I've recently updated my device, so it's probably worth trying again.
Thanks.

**Reply by axel37** [reply]
Update : Nope, still doesn't work

**Comment by theerror** [comment]
I'm afraid that SoloKey is dead project now. Well, seems that ti was big mistake to go for it. Now I would be much smarter next time. It was delivered HW, but without SW it's just useless keyring. I'm making note new experience - do not buy promises, wait for results. Looks, that Nitrokey would be better option. It's pity. It was promising project.

**Reply by msiqx9** [reply]
I strongly disagree with you.  The project seems to be dead but my Solo v2 key continues to work very well.  I've now moved to using Passkeys and the Solo v2 has a generous 100 slots.

**Reply by theerror** [reply]
> I strongly disagree with you. The project seems to be dead but my Solo v2 key continues to work very well. I've now moved to using Passkeys and the Solo v2 has a generous 100 slots.

Yeah, it depends on usecase, indeed. My problem is, that I do need FIDO2 certified device and this i not going to happen, even that it was promised. Unfortunately for me.

**Comment by Yarvid** [comment]
I know this is a rather old post, but I had the same problem with my Nitrokey 3. I also registered the key on my desktop but could not use it to login on my android phone (same error message).

What helped on my end was to deactivate the key in Bitwarden Desktop as 2FA and register it again but from the phone using NFC. After that I had no trouble login in on the phone or the desktop.
